examples/cloud_export_example_test.go

```diff
@@ -14,11 +15,12 @@ import (
 )
 
 func TestCloudExportAPIExample(t *testing.T) {
-	assert.NoError(t, runGRPCCRUDCloudExport())
-	assert.NoError(t, runGRPCGetAllCloudExports())
+	t.Parallel()
```

Comment:
Why do we want to run _examples_ in parallel (or more effectively in any way)? I'd personally prefer to keep things serial, so that the recipient of the demo can understand it.

Reply:
To speed development time.
`t.Parallel()` allows examples to be run in parallel, but providing `-parallel 1` to `go test` will run them sequentially. I added such flag to README.